docs/release_notes/release_1_0_1.rst

```diff
@@ -0,0 +1,33 @@
+Release 1.0.1
+=============
+
+This release is the first one under the name "Lewis". Its main purpose is to make a PyPI package
+available as well as online documentation under the new name.
+
+Nevertheless version 1.0.1 fixes some bugs and introduces a few new features that were originally
+scheduled for release 1.1 but had already been finished at the time of the release.
+
+New features
+------------
+
+ - It is now possible to obtain device interface documentation via the command line
+   and the control server, making it easier to communicate with unfamiliar devices.
+   Thanks to `David Michel`_ for requesting this feature.
+ - Lewis is now available as a `PyPI`_-package and can be installed via ``pip``.
+ - Documentation is now generated via Sphinx and has been made available online on `RTD`_.
+
+Bug fixes and other improvements
+--------------------------------
+
+ - The control server can now be bound to a hostname instead of an IP-address (very useful
+   for ``localhost`` in particular).
+ - pcaspy is now an optional requirement that has to be enabled explicitly in the requirements.txt
+   file or installation via ``pip install lewis[epics]``.
+ - Error messages displayed on the command line have been improved.
+ - A flake8 job has been added to the continuous integration pipeline to enforce Python
+   style guidelines in the codebase.
+
+
+.. _David Michel: https://github.com/dmichel76
+.. _PyPI: https://pypi.python.org/pypi/lewis
+.. _RTD: http://lewis.readthedocs.io/en/latest/
```

Comment:
Thought about this as well and had "stable" first instead of "latest" but that doesn't make sense either. We'd have to add a separate build for each specific release tag in RTD, but that should not be a problem.